### PR TITLE
Send request-id up- and downstream

### PR DESCRIPTION
**Pull request by alex-schneider**

### CODE REVIEW

File: config/env/env.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func DecodeWithPrefix(conf interface{}, prefix string) {
 		envVal = strings.Split(envVal, ",")[0]
 
 		mapVal, exist := envMap[envVal]
-		if !exist || mapVal == "" {
```
Comment (malud) [inline config/env/env.go:60]: Why?
Define an empty env would even override Couper internal defaults on the config object.

Comment (malud) [reply config/env/env.go:60]: `DefaultSettings`

**Review by malud — CHANGES_REQUESTED** [review]

File: config/env/env.go (hunk 1 above)
Comment (alex-schneider) [reply config/env/env.go:60]: It's right - ENV config has priority over other settings.

**Review by malud — CHANGES_REQUESTED** [review]
Builds are failing.

**Review by malud — APPROVED** [review]

**Pull request merged**